packages/cli/test/unit/commands/git.test.ts

```diff
@@ -25,7 +25,7 @@ describe('git', () => {
           id: 'unlinked',
           name: 'unlinked',
         });
-        client.setArgv('projects', 'connect');
```

Comment:
How was this test working before?

Reply:
The test calls `git()` directly, which only looks at the subcommand.